build.go

```diff
@@ -104,60 +104,60 @@ func Build(w io.Writer, config Config, options *BuildOptions) error {
 		bodyStr = bodyStr[:len(bodyStr)-1]
 	}
 
-	_, err := w.Write([]byte(bodyStr))
+	_, err := writer.Write([]byte(bodyStr))
 	return err
 }
 
-//nolint:gocognit
-func buildBlock(sb io.StringWriter, parent *Directive, block Directives, depth int, lastLine int, options *BuildOptions) {
-	for i, stmt := range block {
+//nolint:cyclop
```

Comment:
I think you are adding this because you modified the variable names and now are getting penalized by the linter. I don't know if reverting the variable names avoids that.

There's probably a bit of code in this project that runs afoul of the linter. I think it would be better to address those cases separately rather than disabling linting rules without a good reason. If we're going to disable them, why have them?

Reply:
This is for fixing one of the left-over linter issues related to function complexity and was not introduced because of the new logic changes in this PR. 

Reference for cyclop: https://github.com/bkielbasa/cyclop

But yeah I agree with you, changes for fixing the linter would be better to live in its own separate PR. Thanks for your feedback Stephen!